tools/azure-rest-api-specs-examples-automation/dotnet/test_main.py

```diff
@@ -20,14 +24,15 @@
 using Azure.ResourceManager.ContainerInstance.Models;
 using Azure.ResourceManager.Models;
 using Azure.ResourceManager.Resources;
+using NUnit.Framework;
 
 namespace Azure.ResourceManager.ContainerInstance.Samples
 {
     public partial class Sample_ContainerGroupCollection
     {
         // ContainerGroupsListByResourceGroup
-        [NUnit.Framework.Test]
-        [NUnit.Framework.Ignore("Only verifying that the sample builds")]
+        [Test]
+        [Ignore("Only verifying that the sample builds")]
```

Comment:
I only changes the first method (this case has multiple methods) to show that we currently work with both old style and new style.